### PR TITLE
DOC: Add ``rng.f()`` plot

### PR DESCRIPTION
**Pull request by linus-md**
This PR adds a plot with histogram and pdf to ``rng.f()``

### CODE REVIEW

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -1535,6 +1535,8 @@ cdef class Generator:
         The corresponding probability density function for ``n = 20`` 
         and ``m = 20`` is:
         
+        >>> import matplotlib.pyplot as plt
+        >>> import scipy.stats as stats
```
Comment (rkern) [inline numpy/random/_generator.pyx:1539]:
```suggestion
        >>> from scipy import stats
```

Comment (rkern) [reply numpy/random/_generator.pyx]: Don't forget to add `# doctest: +SKIP` to these `scipy` imports. Same for #25530

Comment (linus-md) [reply numpy/random/_generator.pyx]: @rkern I added the skip in this line as well ``>>> from scipy.stats import binom  # doctest: +SKIP``but now there is an error

Comment (rkern) [reply numpy/random/_generator.pyx]: All of the descendant lines need the `# doctest: +SKIP`, too; it works on a per-line (well, per-`>>>`) basis. This is one of the reasons we try not to use `scipy` in our docstring examples too much.

**Review by rkern — APPROVED** [review]

**Pull request merged**